packages/blade/src/components/Box/__tests__/Box.web.test.tsx

```diff
@@ -96,4 +99,28 @@ describe('<Box />', () => {
     }
     console.error = tempConsoleError;
   });
+
+  it('should support ref on Box', async () => {
```

Comment:
couldn't figure out how I can write test for react native because it doesn't support `addEventListener` and `focus` on Box wouldn't really work as its not button / link.

Reply:
You can just test if the ref isn't null or has the methods like focus or etc?